src/DotNetWorker/Hosting/BootstrapHostBuilder.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Azure.Functions.Worker.Hosting
+{
+    internal class BootstrapHostBuilder : IHostBuilder
```

Comment:
Can you add a summary comment/doc here?

Reply:
Yup. Also included a link to where this came from originally (the aspnetcore repo).